CHANGELOG.unreleased.md

```diff
@@ -16,6 +16,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Backups] Fix MESSAGE_METHOD_UNKOWN(VDI.get_cbt_enabled) on XenServer < 7.3 (PR [#8038](https://github.com/vatesfr/xen-orchestra/pull/8038))
+- [Backup] keep only one snapshot for all the schedules (PR [#8051](https://github.com/vatesfr/xen-orchestra/pull/8051))
```

Comment:
Maybe add more context, like to explain its in case of `snapshot retention` equals to 0 or if no `rolling snapshot` are selected for the backup